### ISSUE TITLE
VisualCodeGrepper on Linux

### ISSUE THREAD

**Issue by rakeshkanyal** [nccgroup/VCG#8]
Can we run VisualCodeGrepper on Linux? I am trying to use it in CentOS which is running on docker container.using WINE but getting issues like X server is not running and $DISPLAY is not set properly.
Can someone please let me know if we can run it on Linux terminal?

**Comment by sjortiz** [comment]
I think that you could use xvfb to run it. maybe it helps.

**Comment by inthenets** [comment]
Yes, a way to run headless on Linux via wine/xvfb or otherwise would be great.
